platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SpaceActivitiesTestIT.java

```diff
@@ -251,4 +251,13 @@ public void test06_CheckSpacesWhenAddmoreThen10Space() {
     spaceManagement.deleteSpace(space12, false);
 
   }
-}
+  @Test
+  @Tag("SOC-6041")
+
+  public void test07_CheckActivityPostOnSpace() {
+    String activity1 = "activity1" + getRandomNumber();
+
+    activityStream.addActivity(activity1, "");
+    activityStream.checkActivity(activity1);
+  }
+  }
```

Comment:
Indentation is not good.

Reply:
I am just talking about changing the code indentation, not the content.

Reply:
The Jira issue was SOC-6041
https://jira.exoplatform.org/browse/SOC-6041?filter=24603

Reply:
I don't talk about the issue, only about the indentation of the code (to make the code more readable).

Reply:
It's clear ok.

Reply:
I have added Infos . 
![Screenshot from 2019-06-26 18-15-15](https://user-images.githubusercontent.com/50202793/60196859-687e1b00-983e-11e9-8223-f292ac5e736f.png)